### DISCUSSION TITLE
ipxe requesting kernel from tftp then nothing

### DISCUSSION THREAD

**Discussion by mooperd** [ipxe/ipxe#1167] [Q&A]
Hello,

I'm struggling to get PXE booting working. The kernel is being requested from TFTP but its just slently failing. I can see in the TFTP logs everything upto the point of requesting the kernel but then nothing. Its not trying to load the initrd. I'm using the latest version of proxmox to play with all of this.

Please can I have a little help debugging this? My google-fu is failing me.

Thanks,

Andrew

/tftp/boot/pxelinux.cfg/default
`
default menu.c32
menu title Ubuntu installer

LABEL jammy
  MENU LABEL Install Ubuntu Jammy (22.04)
  KERNEL vmlinuz-5.15.0-97-generic
  INITRD initrd.img-5.15.0-97-generic
  APPEND ip=dhcp auto=true  

prompt 0
timeout 300
`
Some logs.

`
Mar  4 09:21:43 meowtronic dhcpd[4760]: DHCPDISCOVER from bc:24:11:3a:ec:3a via ens19
Mar  4 09:21:44 meowtronic dhcpd[4760]: ns1.example.org: host unknown.
Mar  4 09:21:44 meowtronic dhcpd[4760]: ns2.example.org: host unknown.
Mar  4 09:21:44 meowtronic dhcpd[4760]: DHCPOFFER on 10.141.0.100 to bc:24:11:3a:ec:3a via ens19
Mar  4 09:21:44 meowtronic dhcpd[4760]: DHCPDISCOVER from bc:24:11:3a:ec:3a via ens19
Mar  4 09:21:44 meowtronic dhcpd[4760]: DHCPOFFER on 10.141.0.100 to bc:24:11:3a:ec:3a via ens19
Mar  4 09:21:46 meowtronic dhcpd[4760]: DHCPREQUEST for 10.141.0.100 (10.141.255.254) from bc:24:11:3a:ec:3a via ens19
Mar  4 09:21:46 meowtronic dhcpd[4760]: DHCPACK on 10.141.0.100 to bc:24:11:3a:ec:3a via ens19
Mar  4 09:21:46 meowtronic tftpd[7648]: tftpd: trying to get file: boot/pxelinux.0
Mar  4 09:21:46 meowtronic tftpd[7648]: tftpd: serving file from /tftp
Mar  4 09:21:46 meowtronic tftpd[7650]: tftpd: trying to get file: boot/ldlinux.c32
Mar  4 09:21:46 meowtronic tftpd[7650]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7652]: tftpd: trying to get file: boot/pxelinux.cfg/01-bc-24-11-3a-ec-3a
Mar  4 09:21:47 meowtronic tftpd[7652]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7654]: tftpd: trying to get file: boot/pxelinux.cfg/0A8D0064
Mar  4 09:21:47 meowtronic tftpd[7654]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7656]: tftpd: trying to get file: boot/pxelinux.cfg/0A8D006
Mar  4 09:21:47 meowtronic tftpd[7656]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7658]: tftpd: trying to get file: boot/pxelinux.cfg/0A8D00
Mar  4 09:21:47 meowtronic tftpd[7658]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7660]: tftpd: trying to get file: boot/pxelinux.cfg/0A8D0
Mar  4 09:21:47 meowtronic tftpd[7660]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7662]: tftpd: trying to get file: boot/pxelinux.cfg/0A8D
Mar  4 09:21:47 meowtronic tftpd[7662]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7664]: tftpd: trying to get file: boot/pxelinux.cfg/0A8
Mar  4 09:21:47 meowtronic tftpd[7664]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7666]: tftpd: trying to get file: boot/pxelinux.cfg/0A
Mar  4 09:21:47 meowtronic tftpd[7666]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7668]: tftpd: trying to get file: boot/pxelinux.cfg/0
Mar  4 09:21:47 meowtronic tftpd[7668]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7670]: tftpd: trying to get file: boot/pxelinux.cfg/default
Mar  4 09:21:47 meowtronic tftpd[7670]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7672]: tftpd: trying to get file: boot/menu.c32
Mar  4 09:21:47 meowtronic tftpd[7672]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7674]: tftpd: trying to get file: boot/libutil.c32
Mar  4 09:21:47 meowtronic tftpd[7674]: tftpd: serving file from /tftp
Mar  4 09:21:47 meowtronic tftpd[7676]: tftpd: trying to get file: boot/pxelinux.cfg/default
Mar  4 09:21:47 meowtronic tftpd[7676]: tftpd: serving file from /tftp
Mar  4 09:21:55 meowtronic tftpd[7678]: tftpd: trying to get file: boot/vmlinuz-5.15.0-97-generic
Mar  4 09:21:55 meowtronic tftpd[7678]: tftpd: serving file from /tftp
Mar  4 09:21:58 meowtronic tftpd[7680]: tftpd: trying to get file: boot/vmlinuz-5.15.0-97-generic
Mar  4 09:21:58 meowtronic tftpd[7680]: tftpd: serving file from /tftp
Mar  4 09:21:58 meowtronic tftpd[7682]: tftpd: trying to get file: boot/vmlinuz-5.15.0-97-generic

`

<img width="1318" alt="image" src="https://github.com/ipxe/ipxe/assets/3006149/5029e8fd-efee-4161-bec2-b462fd6a3d1a">

**Comment by NiKiZe** [comment]
You seem to be using pxelinux, not iPXE.

But general advice, tcpdump or wireshark to see what is going on on the wire, possible guess is MTU issue, but that is a wild guess.
